doc/control.rst

```diff
@@ -108,7 +108,6 @@ Control system synthesis
 .. autosummary::
    :toctree: generated/
 
-    acker
```

Comment:
As mentioned above, suggest we leave `acker` as a publicly visible function.

Reply:
I reverted this change :) 

As I said above, I think it is a little bit misleading to a several entry points for a single functionality. It reduces the redundancy.